lib/Github/Api/Organization/Teams.php

```diff
@@ -95,16 +95,49 @@ public function removeMember($team, $username, $organization)
         return $this->delete('/orgs/'.rawurlencode($organization).'/teams/'.rawurlencode($team).'/memberships/'.rawurlencode($username));
     }
 
-    public function repositories($team)
+    /**
+     * List a team's repositories.
+     *
+     * @link https://docs.github.com/en/rest/teams/teams#list-team-repositories
+     *
+     * @param string $team         team ID or slug of the team name
+     * @param string $organization organization name
+     * @param array  $params       list of extra parameters.
+     *
+     * @return array
+     */
+    public function repositories($team, $organization = '', $params = [])
     {
-        return $this->get('/teams/'.rawurlencode($team).'/repos');
+        $parameters = array_merge([
+            'page' => 1,
+            'per_page' => 30,
+        ], $params);
```

Comment:
This should not be needed as the [`ResultPager` class](https://github.com/KnpLabs/php-github-api/blob/aed01678b5a005b7430d8cb952985113177947fe/lib/Github/ResultPager.php) should be used to loop over multiple pages of results